### PR TITLE
[api][SIMS #1188]FT Ecert File Issue

### PR DESCRIPTION
**Pull request by andrewsignori-aot**
- Added COE status as a constraint to consider the disbursement ready. So far, only after the COE is completed a document number will be generated and the disbursement should be considered ready;
- Filtered grants that have 0(zero) dollar amounts. These grants should be present on DB but should not be part of the e-Cert file;
- Improved the comments on the method to be clear the list of the current requirements for a disbursement happens.
image

### CODE REVIEW

**Review by guru-aot — APPROVED** [review]
LGTM

File: sources/packages/api/src/services/disbursement-schedule-service/disbursement-schedule-service.ts [hunk 1]
```diff
@@ -156,11 +156,14 @@ export class DisbursementScheduleService extends RecordDataModelService<Disburse
 
   /**
    * Get all records that must be part of the e-Cert files and that were not sent yet.
-   * Consider any record that is scheduled in upcoming days or in the past.
-   * Check if the student has a valid SIN.
-   * Consider only completed Student Applications with signed MSFAA date.
-   * Check if there are restrictions applied to the student account that would
-   * prevent the disbursement.
+   * Criteria to be a valid disbursement to be send.
```
Comment (henryoforeh-aot) [inline sources/packages/api/src/services/disbursement-schedule-service/disbursement-schedule-service.ts:159]: Please change "send" to "sent".

**Review by henryoforeh-aot — APPROVED** [review]
Approved, just a minor comment.

**Review by dheepak-aot — APPROVED** [review]
LGTM.

**Review by ann-aot — APPROVED** [review]
👍

File: sources/packages/api/src/services/disbursement-schedule-service/disbursement-schedule-service.ts (hunk 1 above)
Comment (andrewsignori-aot) [reply sources/packages/api/src/services/disbursement-schedule-service/disbursement-schedule-service.ts]: Changed!

**Review by henryoforeh-aot — APPROVED** [review]

**Review by ann-aot — COMMENTED** [review]
👍

**Pull request merged**